src/components/Banner/banner.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styles from './banner.module.scss';
+
+export function Banner() {
+    const [displayBanner, setDisplayBanner] = React.useState(true);
+
+    const display = displayBanner ? styles.banner : styles.hidden;
+
+    function toggleBanner() {
+        setDisplayBanner((prevState) => !prevState);
+    }
+
+    return (
+        <div className={display}>
+            <div>
+                <h3 className={styles.textBold}>Method Conference</h3>
+            </div>
+            <div className={styles.textArea}>
+                <p className={styles.textBold}>Join us for a full day of code & learning.</p>
+                <p className={styles.text}>Saturday, October 12, 2024, @ eFactory</p>
+            </div>
+            <div className={styles.container}>
+                <a href="https://www.methodconf.com/2024/" target="_blank">
+                    <button className={styles.button}>BUY TICKETS</button>
+                </a>
+                <button className={styles.exitButton} onClick={toggleBanner}>
+                    <img src="/images/icons/exit-cross.png" className={styles.exit} alt="close banner" />
```

Comment:
Do we have any kind of icon system like font awesome in this project? If not can you add a eslint disable comment for this warning so we don't keep setting it?

Reply:
We do not have any icon systems on this project, so I took the img from the figma file. For some reason, the typical comment syntax of /* eslint-disable */ and /* eslint-enable */ does not seem to function. Instead, the comments show up as plan text on the website.

Reply:
Yeah react has their own comment format in JSX you'll probably need to do something like what's in these examples

https://stackoverflow.com/questions/30766441/how-to-use-comments-in-react

Reply:
That helps a lot! I've added the disable comments, as well as comments to document why they were placed there and when they should be removed.